src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,68 @@
+package edu.kis.vh.nursery;
+
+/**
+Ta klasa służy do tworzenia tablicy i pracy z danymi tej tablicy.
+ */
+public class DefaultCountingOutRhymer {
+    /**Maksymalny rozmiar tablicy*/
+    public static final int CAPACITY = 12;
+
+    /**Zmienna, która oznacza pustą tablicę*/
+    public static final int EMPTY = -1;
+
+    /**Tworzenie tablicy z maksymalnym rozmiarem*/
+    private int[] numbers = new int[CAPACITY];
+
+    /**Tworzenie iteratora dla tablicy z początkowym znaczeniem EMPTY (-1)*/
+    public int total = EMPTY;
+
+    /**
+     funkcja która dodaje podany element do tablicy
+     Jeśli tablica nie jest pelna, to element będzie dodany do tablicy i iterator będzie zwiększony
+     Jeśli tablica jest pełna, funkcja nie podejmie żadnych działań
+     */
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    /**
+     Funkcja sprawdza, czy jest tablica pusta.
+     Jeśli tak to zwraca true, jeśli nie false
+     */
+    public boolean callCheck() {
+        return total == EMPTY;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods.